Electron/README.md

```diff
@@ -0,0 +1,118 @@
+# Electron debugging (main and renderer process)
+
+by [Kenneth Auchenberg](https://twitter.com/auchenberg)
+
+This recipe shows how to use the built-in Node Debugger and the [Debugger for Chrome](https://github.com/Microsoft/vscode-chrome-debug) extension with VS Code to debug [Electron](https://electron.atom.io) applications.
+
+[Electron](https://electron.atom.io) consists of two proceses: A Main process running in NodeJS and a render process in Chromium. This means that you'll need to use two debugger instances within VS Code to debug both processes. This is the reason for why you both the built-in Node Debugger and the [Debugger for Chrome](https://github.com/Microsoft/vscode-chrome-debug).
+
+## Getting Started
+
+1. Make sure to have the latest version of VS Code installed.
+
+2. Make sure to the latest version of [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome) extension installed in VS Code.
+
+3. This guide assumes that you are using the [electron-Quick-start project](https://github.com/electron/electron-quick-start). Clone the repo to get started
+    > 
+    ```
+    git clone git@github.com:electron/electron-quick-start.git
+    cd electron-quick-start
+    npm install
+    code .
+    ```
+
+## Configure launch.json File
+
+- Click on the Debugging icon in the Activity Bar to bring up the Debug view.
+Then click on the gear icon to configure a launch.json file, selecting **Chrome** for the environment:
+
+   ![configure_launch](configure_launch.png)
+
+- Replace content of the generated launch.json with the following two configurations:
+
+  ```json
+    {
+        "version": "0.2.0",
+        "configurations": [
+            {
+                "type": "node",
+                "request": "launch",
+                "name": "Electron: Main",
+                "protocol": "inspector",
```

Comment:
Worth mentioning somewhere that this depends on the Electron version? I suppose in this tutorial they'll get the latest one

Reply:
Feedback incorporated.